File: docs/fixtures/core/documents/missing_field/rust/calculate_velocity.rs
/// 计算平均速度
///
/// # Arguments
/// - distance_m：行进距离
/// - duration_s：持续时间
/// # Returns
/// - 平均速度
pub fn calculate_velocity(distance_m: f64, duration_s: f64) -> f64 {
    let velocity_m_per_s = distance_m / duration_s;
    velocity_m_per_s
}

File: docs/fixtures/core/documents/missing_field/python/calculate_velocity.py
def calculate_velocity(distance_m: float, duration_s: float) -> float:
    """
    计算平均速度

    Args:
        distance_m: 行进距离
        duration_s: 持续时间
    Returns:
        float: 平均速度
    """
    velocity_m_per_s = distance_m / duration_s
    return velocity_m_per_s
